src/renderer/components/watch-video-comments/watch-video-comments.vue

```diff
@@ -200,6 +212,12 @@
                 >
               </template>
             </router-link>
+            <p
+              v-if="isReplyHighlighted(comment, reply)"
+              class="replyHighlighted"
+            >
+              {{ $t("Highlighted reply") }}
```

Comment:
Refer to all new code with new translation entry:
- Missing new entries in `static/locales/en-US.yaml`
- The key probably should be under `Comments` like `Comments.Pinned by` in the same file